qcodes/tests/test_abstract_instrument.py

```diff
@@ -138,9 +139,11 @@ def test_unit_value_error():
     """
     with pytest.raises(ValueError, match="This is inconsistent with the unit defined"):
         VoltageSourceBadUnit("driver3")
-    assert not VoltageSourceBadUnit.instances()
 
+    # assert not VoltageSourceBadUnit.instances()
```

Comment:
can't this be left in as the original intent of the test, now that it's skipped?

Reply:
This test is not skipped ;) Since the core of the test still works correctly

Reply:
Why apart from this check the test passes and mostly tests the functionality that is not reverted

Reply:
why - so that this test is also preserved as is, and is executed, but it's failure won't influence the test summary. however, as you see appropriate :)

Reply:
Lets split it into two to make this more clear